### DISCUSSION TITLE
Add an option for non-error exit status mark + customization

### DISCUSSION THREAD

**Discussion by eadmaster** [akinomyoga/ble.sh#150] [Q&A] [answered]
I'd like to see the exit status even when successful, possibly with a green color
` [ble: exit 0]`

Also, i'd like to add an emoji to both error and non-error exits to make them more readable.

Example made via PS1 customization:
![Screenshot_20211112_223553](https://user-images.githubusercontent.com/925171/141537528-3bc01703-92f6-4b88-aee9-bd69830e4f2d.png)

**Comment by akinomyoga** [comment, marked as answer]
If you want to print the exit status for both cases of successful execution and failure, you may turn off the exit-status display from ble.sh by setting an empty string to [`bleopt exec_errexit_mark`](https://github.com/akinomyoga/ble.sh/wiki/Manual-%C2%A74-Editing#user-content-bleopt-exec_errexit_mark) and instead print one by yourself. You can print your own exit-status information via whatever you like such as [`blehook POSTEXEC`](https://github.com/akinomyoga/ble.sh/wiki/Manual-%C2%A71-Introduction#122-function-blehook-hook_nameshell-command-v04-set-hooks), `PS1`, or `PROMPT_COMMAND`. Here's an example using `blehook POSTEXEC`:

```bash
# blerc

# Turn off the exit status [ble: exit $?] by ble.sh
bleopt exec_errexit_mark=

# Print status
print-exit-status() {
  local exit=$?
  if ((exit==0)); then
    echo $'status: \e[32m0 ♥\e[m'
  else
    echo $'status: \e[31m'"$exit"$' ♦\e[m'
  fi
}
blehook POSTEXEC=print-exit-status
```

**Comment by eadmaster** [comment]
thank you, it is working as expected, i've made some changes to better suits my tastes (that's some tricky code to edit):
````
# Turn off the exit status [ble: exit $?] by ble.sh
bleopt exec_errexit_mark=

# Print status
print-exit-status() {
  local exit=$?
  if ((exit==0)); then
    echo $'\e[2mstatus: \033[2;32m'"$exit"$' \u2665\e[m'
  else
    echo $'\e[2mstatus: \033[2;31m'"$exit"$' \u1F615\e[m'
    
  fi
}
blehook POSTEXEC=print-exit-status
````
